src/libultra/rmon/sprintf.c

```diff
@@ -1,4 +1,5 @@
 #include "global.h"
```

Comment:
This doesn't look to be libultra's implementation of sprintf, so should probably move it out?

Reply:
Oh, nice catch.
I think I would prefer to keep this PR to headers only. We can fix it in the future